Frontend/Gmfctn/src/app/modules/modal-windows/saythank-modal/saythank-modal.component.ts

```diff
@@ -0,0 +1,110 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MatDialog, MatDialogRef, MAT_DIALOG_DATA} from '@angular/material/dialog';
+import { LeaveTheCommentComponent } from '../leave-the-comment/leave-the-comment.component';
+import '../../../models/user';
+import '../../../models/achievement';
+@Component({
+  selector: 'app-saythank-modal',
+  templateUrl: './saythank-modal.component.html',
+  styleUrls: ['./saythank-modal.component.scss']
+})
+export class SaythankModalComponent implements OnInit {
```

Comment:
it is incorrect naming. it should be at least SayThankModal, file name should be say-thank-modal.componets.ts and use same rules for selector and scss, spec.ts and html files.